.uberenv_config.json

```diff
@@ -9,7 +9,7 @@
 "spack_packages_path": ["scripts/spack/radiuss-spack-configs/packages", "scripts/spack/packages"],
 "spack_concretizer": "clingo",
 "vcpkg_url": "https://github.com/microsoft/vcpkg",
-"vcpkg_commit": "6f7ffeb18f99796233b958aaaf14ec7bd4fb64b2",
+"vcpkg_commit": "52b52b4588b916152ff662f97842e03db83fa5d1",
```

Comment:
Please update the `RELEASE_NOTES` to mention the vcpkg update and to point to this version.

Reply:
done.

Reply:
Done.